<!-- Dao introduction -->
	  <div class="introduction">
	    <table width="840" border="0" align="center">
		        <tr>
		          <td width="166"><img src="images/dfy.jpg" alt="" width="166" height="192" style="padding-right:10px"/></td>
		          <td><h2>Fuying Dao</h2>
		            <p align="justify">Hi, I am Fuying Dao, welcome to my personal page! I am now a postdoctoral fellow of School of Biological Sciences at Nanyang Technological University.</p>
		            <p><strong>Google Scholar: </strong><a href="https://scholar.google.com/citations?user=Htl60dYAAAAJ&hl=en"> Fuying Dao</a></p>
			    <p><strong>ORCID:</strong><a href="https://orcid.org/0000-0001-5285-6044"> 0000-0001-5285-6044</a></p>
			    <p><strong>E-mail: </strong><a href="mailto:fuying.dao@ntu.edu.sg">fuying.dao@ntu.edu.sg</a></p>
			    <br>
			  </td>
	                </tr>
		        <tr>
		          <td  colspan="2">
				  <h2><strong>Education</strong></h2>
				    <p>Ph.D in Biomedical Engineering (adviser: <a href="http://lin-group.cn/lin.html">Dr. Hao Lin</a>), University of Electronic Science and Technology of China, 09/2019 - 06/2023</p>
				    <p>M.S in Bioinformatics (adviser: <a href="http://lin-group.cn/lin.html">Dr. Hao Lin</a>), University of Electronic Science and Technology of China, 09/2016 - 06/2019</p>
				    <p>B.S in Biotechnology (adviser: <a href="http://lin-group.cn/lin.html">Dr. Hao Lin</a>), University of Electronic Science and Technology of China, 09/2012 - 06/2016</p>
				  <h2><strong>Academic Experience</strong></h2>
				    <p>Postdoctoral Scholar (adviser: <a href="https://fullwoodlab.com/">Dr. Melissa Jane Fullwood</a>), School of Biological Sciences, Nanyang Technological University, 08/2023 - now</p>
				    <p>Visiting Scholar (adviser: <a href="https://fullwoodlab.com/">Dr. Melissa Jane Fullwood</a>), School of Biological Sciences, Nanyang Technological University, 10/2021 – 10/2022</p>
				<h2><strong>Honours and Awards</strong></h2> 
				    <p>The <a href="https://www.ntu.edu.sg/research/research-careers/the-eric-and-wendy-schmidt-ai-in-science-postdoctoral-fellowship">Eric and Wendy Schmidt AI in Science Postdoctoral Fellowship</a>, Nanyang Technological University, 2023</p>
				    <p>The <a href="https://www.csc.edu.cn/">China Scholarship Council (CSC) Scholarship</a>, University of Electronic Science and Technology of China, 2021</p>
				<h2><strong>Research interests</strong></h2> 
				    <p>My primary interests are in utilizing high-throughput sequencing data (eg. Hi-C and ChIA-PET) and deep learning algorithms to uncover epigenetic mechanisms associated with disease processes:</p>
				  <p>*Method development -- How to identify new biological insights from large amounts of available RNA-Seq data? -- Develop deep learning architecture to predict chromatin interactions of large cohorts of cell lines or clinical samples from RNA-Seq data</p>
				  <p>*The 3D structure of the genome -- Explore differences in the chromatin interactions of different samples to understand how chromatin interactions may be important for the identification of diagnostic and predictive biomarkers for epigenetically driven cancers</p>
				  <h2><strong>Publications</strong></h2>
				  	<p>13. <strong>Fuying Dao&</strong>, Xueqin Xie &, Hongqi Zhang &, Zhengxing Guan, Changchun Wu, Wei Su, Hongyan Lai, Dong Gao, Yuhe Yang, Yang Zhang, Lin Ning, Shi-Hao Li, Yuduo Hao, Benjamin Lebeau, Crystal Ling Chia Yin, Jian Huang, Melissa Jane Fullwood*, Hao Lin*, Hao Lv*. (2024) PlantEMS: A Comprehensive Database of Epigenetic Modification Sites Across Multiple Plant Species. <em><strong>Plant Communications</strong></em>,101228. (2023 IF: 9.4) <a href="https://www.sciencedirect.com/science/article/pii/S2590346224006497">[Link]</a></p>
				  	<p>12. Yuduo Hao, Kaiyuan Han, Ting Wang, Junwen Yu, Hui Ding*, <strong>Fuying Dao*</strong>. (2024) Exploring the potential of epigenetic clocks in aging research. <em><strong>Methods</strong></em>, 231: 37-44. (2023 IF: 4.2) <a href="https://www.sciencedirect.com/science/article/abs/pii/S1046202324001919">[Link]</a></p>
					<p>11. <strong>Fu-Ying Dao</strong>, Meng-Lu Liu, Wei Su, Hao Lv, Zhao-Yue Zhang, Hao Lin*, Li Liu*. (2023) AcrPred: A hybrid optimization with enumerated machine learning algorithm to predict Anti-CRISPR proteins. <em><strong>International Journal of Biological Macromolecules</strong></em>, 228: 706-714. (2021 IF: 8.025) <a href="https://www.sciencedirect.com/science/article/pii/S0141813022031671">[Link]</a></p>
					<p>10. <strong>Fu-Ying Dao</strong>, Hao Lv, Melissa J. Fullwood*, Hao Lin*. (2022) Accurate Identification of DNA Replication Origin by Fusing Epigenomics and Chromatin Interaction Information.  <em><strong>Research</strong></em>, 2022: ID 9780293. (2021 IF: 11.036) <a href="https://spj.science.org/doi/10.34133/2022/9780293">[Link]</a></p>
					<p>9. <strong>Fu-Ying Dao</strong>, Hao Lv, Zhao-Yue Zhang, Hao Lin*. (2021) BDselect: a package for k-mer selection based on binomial distribution. <em><strong>Current Bioinformatics</strong></em>, 17(3): 238-244(7). (2021 IF: 4.850) <a href="https://www.ingentaconnect.com/content/ben/cbio/2022/00000017/00000003/art00003">[Link]</a></p>
					<p>8. <strong>Fu-Ying Dao</strong>, Hao Lv, Wei Su, Zi-Jie Sun, Qin-Lai Huang, Hao Lin* (2021) iDHS-Deep: An integrated tool for predicting DNase I hypersensitive sites by deep neural network. <em><strong>Briefings in Bioinformatics</strong></em>, 22(5): bbab047. (2020 IF: 11.622) <a href="https://academic.oup.com/bib/article-abstract/22/5/bbab047/6158360?redirectedFrom=fulltext">[Link]</a></p>
					<p>7. Hao Lv&, <strong>Fu-Ying Dao</strong>&, Hasan Zulfiqar, Wei Su, Hui Ding, Li Liu*, Hao Lin* (2021) A sequence-based deep learning approach to predict CTCF-mediated chromatin loop. <em><strong>Briefings in Bioinformatics</strong></em>, 22(5): bbab031. (2020 IF: 11.622) <a href="https://academic.oup.com/bib/article/22/5/bbab031/6149346?login=true">[Link]</a></p>
					<p>6. <strong>Fu-Ying Dao</strong>, Hao Lv, Dan Zhang, Zi-Mei Zhang, Li Liu*, Hao Lin*. (2021) DeepYY1: a deep learning approach to identify YY1-mediated chromatin loops. <em><strong>Briefings in Bioinformatics</strong></em>, 22(4):bbaa356. (2020 IF: 11.622) <a href="https://academic.oup.com/bib/article/22/4/bbaa356/6024741?login=true">[Link]</a> (<a href="images/ESI-DeepYY1.png" style="text-decoration:none"><span style="color: red">ESI</span></a>)</p>
					<p>5. <strong>Fu-Ying Dao</strong>, Hao Lv, Hasan Zulfiqar, Hui Yang, Wei Su, Hui Gao, Hui Ding, Hao Lin*. (2021) A computational platform to identify origins of replication sites in eukaryotes. <em><strong>Briefings in Bioinformatics</strong></em>, 22(2): 1940–1950. (2020 IF: 11.622) <a href="https://academic.oup.com/bib/article/22/2/1940/5739183?login=true">[Link]</a> (<a href="images/ESI_iORI-Euk.png" style="text-decoration:none"><span style="color: red">ESI</span></a>)</p>
					<p>4. <strong>Fu-Ying Dao</strong> &, Hao Lv &, Yu-He Yang, Hasan Zulfiqar, Hui Gao, Hao Lin*. (2020) Computational identification of N6-Methyladenosine sites in multiple tissues of mammals. <em><strong>Computational and Structural Biotechnology Journal</strong></em>, 18: 1084–1091. (2019 IF: 6.018) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229270/">[Link]</a></p>
					<p>3. <strong>Fu-Ying Dao</strong>, Hao Lv, Fang Wang, Chao-Qin Feng, Hui Ding*, Wei Chen*, Hao Lin*. (2019) Identify origin of replication in Sccharomyces cerevisiae using two-step feature selection technique. <em><strong>Bioinformatics</strong></em>, 35(12):2075-2083. (2018 IF: 4.531) <a href="https://academic.oup.com/bioinformatics/article/35/12/2075/5182294?login=true">[Link]</a> (<a href="images/ESI_iORI-PseKNC.png" style="text-decoration:none"><span style="color: red">ESI,Hot</span></a>)</p>
					<p>2. <strong>Fu-Ying Dao</strong>, Hao Lv, Fang Wang, Hui Ding*. (2018) Recent advances on the machine learning methods in identifying DNA replication origins in eukaryotic genomics. <em><strong>Frontiers in Genetics</strong></em>, 9: 613. (2017 IF: 4.151) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6295579/">[Link]</a></p>
					<p>1. <strong>Fu-Ying Dao</strong>, Hui Yang, Zhen-Dong Su, Wuritu Yang, Yun Wu, Ding Hui, Wei Chen*, Hua Tang*, Hao Lin*. (2017) Recent advances in conotoxin classification by using machine learning methods. <em><strong>Molecules</strong></em>, 22: 1057. (2016 IF: 2.861) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6152242/">[Link]</a></p>
					<p style="color: grey">More details can find in my <a href="https://scholar.google.com/citations?user=Htl60dYAAAAJ&hl=en">Google Scholar</a></p>
				  <h2><strong>GitHub Projects</strong></h2>
				  <p><a href="https://github.com/DaoFuying/iORI-Epi">iORI-Epi</a>: a computational approach to predict replication origin sites (ORIs) using the epigenomic marks, DNA motifs, and chromatin loops</p>
				  <p><a href="https://github.com/DaoFuying/iORI-Euk">iORI-Euk</a>: a computational platform to identify origins of replication sites in eukaryotes</p>
				  <p><a href="https://github.com/DaoFuying/DeepYY1" >DeepYY1</a>: a deep learning approach to identify YY1-mediated chromatin loops</p>
				  <p><a href="https://github.com/DaoFuying/AcrPred">AcrPred</a>: a hybrid optimization with enumerated machine learning algorithm to predict Anti-CRISPR proteins</p>
				  <p><a href="https://github.com/DaoFuying/BDselect">BDselect</a>: a package for k-mer selection based on binomial distribution</p>
				  <h2><strong>Conferences</strong></h2>
				  <p><strong>The 11th International Symposium on 3D Genomics</strong>, 29 Oct 2024, Sanya, China (oral).</p>
				  <p><strong>2024 International Conference on Biomedicine and AI</strong>, 25 Oct 2024, Shanghai, China (oral).</p>
				  <p><strong>AI4Sci / NTCI Conference</strong>, 23-24 July 2024, Singapore (poster).</p>
				  <p><strong>EMBO Workshop Chromatin biology in cancer</strong>, 23-26 April 2024, Singapore (poster).</p>
				  <p><strong>The first regional convening meeting for the Eric and Wendy Schmidt AI in Science Postdoctoral Fellowship</strong>, 23 Feb 2024, Singapore (oral).</p>
				  <br>
				  <br>

	            </tr>

        </table>	
	  </div>
	  <!-- introduction end-->
